modules/Runtime.jsm

```diff
@@ -116,6 +116,15 @@ this.Runtime = {
     appWindow.addEventListener('close', onTargetClose);
   },
 
+  closeDevTools(target) {
+    // TODO: Keep an object in memory that maps windows => opened Dev Tools?
+    // Or query the active window and close the Dev Tools if it's already open?
+    // (I assume the latter. What's the best way to go about this?)
```

Comment:
Hey @cvan, I wasn't sure if you were waiting on me for more input into this PR, so I took another look and realized you're probably waiting for me to answer this question! Sorry about the delay.

I'd keep a WeakMap of target -> devtools window, then query the WeakMap for the devtools window corresponding to the given target here. Querying the active window should also work, although in that case you still need to identify the devtools window associated with that window.